src/NLog.StructuredLogging.Json/Helpers/Sha1Hasher.cs

```diff
@@ -14,17 +14,10 @@ public static string Hash(string input)
 
             byte[] hash;
 
-#if NET452
             using (var sha1 = new SHA1Managed())
             {
                 hash = sha1.ComputeHash(Encoding.UTF8.GetBytes(input));
             }
-#else
-            using (var sha1 = SHA1.Create())
```

Comment:
We should use this one if we're dropping one of the two, as `SHA1` is the "better" of the two as it can leverage the OS crypto libraries if available, if it isn't the `SHA` instance returned will probably be an instance of `SHA1Managed` anyway.

Reply:
So does "Managed" in this sense mean "written in C#", whereas the other one means "you choose"?
 I assume that the output is identical - hashes are well-defined, but the choice is performance?